src/main/java/software/amazon/sns/AmazonSNSExtendedClientBase.java

```diff
@@ -1,21 +1,13 @@
 package software.amazon.sns;
```

Comment:
As mentioned on  all the java docs should be updated to fully match current java docs in SDK V2.